app/controllers/users_controller.rb

```diff
@@ -3,18 +3,20 @@ class UsersController < ApplicationController
 	before_action :set_user, only: [:show, :update, :destroy, :update_password]
 
 	def index
-		@users = authorize current_user
-    render json: @users
+		@users = current_user.admin? ? User.all : current_user
+    render json: {status: {code: 200, message: 'Success.'}, data: UserSerializer.new(@users).serializable_hash[:data]}
   end
 
   def show
-    render json: @user if authorize @user
+    if authorize @user
+      render json: {status: {code: 200, message: 'Success.'}, data: UserSerializer.new(@user).serializable_hash[:data][:attributes]}
```

Comment:
@josephelias-rubyians  Follow the rubocop style guide and see how to write a long line in multiple lines

Reply:
This is applicable for all the similar other lines